tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "module": "commonjs",
     "moduleResolution": "node",
     "noImplicitAny": true,
-    "lib": ["es2022"],
+    "lib": ["es2022", "dom"],
```

Comment:
I hadn't thought we'd need dom as we're not running our code in a browser.  Is this an Athena requirement?

Reply:
Good spot, this was a change I had to put in to stop VSCode constantly flagging errors, I will take it out and check if the errors persist

Reply:
Addressed by removing dom